src/resolvers/Mutation/removeComment.ts

```diff
@@ -48,36 +50,33 @@ export const removeComment: MutationResolvers["removeComment"] = async (
     );
   }
 
-  // Checks whether currentUser with _id === context.userId is not the creator of comment.
-  if (comment.creator.toString() !== context.userId.toString()) {
+  // Checks whether currentUser with _id === context.userId is not the creator of comment or the user is a SUPERADMIN
+  if (
+    comment.creator.toString() !== context.userId.toString() &&
```

Comment:
Shouldn't the organization admin be able to delete comments on a post that corresponds to their organization?

Reply:
Yes! Will update the resolver to allow the same.